go.mod

```diff
@@ -26,6 +26,7 @@ require (
 	github.com/multiformats/go-multiaddr v0.0.1
 	github.com/multiformats/go-multiaddr-dns v0.0.2
 	github.com/multiformats/go-multistream v0.0.1
+	github.com/satori/go.uuid v1.2.0 // indirect
```

Comment:
can you `go mod tidy` to get rid of this indirect import?

Reply:
```
→ go mod tidy -v
go: finding github.com/ipfs/go-datastore/autobatch latest
Fetching https://github.com?go-get=1
Parsing meta tags from https://github.com?go-get=1 (status code 200)

→ cat go.mod
module github.com/libp2p/go-libp2p-kad-dht

require (
        github.com/gogo/protobuf v1.2.1
        github.com/hashicorp/golang-lru v0.5.1
        github.com/ipfs/go-cid v0.0.1
        github.com/ipfs/go-datastore v0.0.1
        github.com/ipfs/go-ipfs-util v0.0.1
        github.com/ipfs/go-log v0.0.1
        github.com/ipfs/go-todocounter v0.0.1
        github.com/jbenet/go-context v0.0.0-20150711004518-d14ea06fba99
        github.com/jbenet/goprocess v0.0.0-20160826012719-b497e2f366b8
        github.com/libp2p/go-libp2p v0.0.2
        github.com/libp2p/go-libp2p-crypto v0.0.1
        github.com/libp2p/go-libp2p-host v0.0.1
        github.com/libp2p/go-libp2p-kbucket v0.0.1
        github.com/libp2p/go-libp2p-net v0.0.1
        github.com/libp2p/go-libp2p-peer v0.0.1
        github.com/libp2p/go-libp2p-peerstore v0.0.1
        github.com/libp2p/go-libp2p-protocol v0.0.1
        github.com/libp2p/go-libp2p-record v0.0.1
        github.com/libp2p/go-libp2p-routing v0.0.1
        github.com/libp2p/go-libp2p-swarm v0.0.1
        github.com/libp2p/go-testutil v0.0.1
        github.com/mr-tron/base58 v1.1.0
        github.com/multiformats/go-multiaddr v0.0.1
        github.com/multiformats/go-multiaddr-dns v0.0.2
        github.com/multiformats/go-multistream v0.0.1
        github.com/satori/go.uuid v1.2.0 // indirect
        github.com/stretchr/testify v1.3.0
        github.com/whyrusleeping/base32 v0.0.0-20170828182744-c30ac30633cc
        golang.org/x/xerrors v0.0.0-20190212162355-a5947ffaace3
)

→ go mod why github.com/satori/go.uuid
go: finding github.com/ipfs/go-datastore/autobatch latest
# github.com/satori/go.uuid
github.com/libp2p/go-libp2p-kad-dht
github.com/ipfs/go-datastore
github.com/satori/go.uuid

```

Reply:
`go mod tidy` adds this line

Reply:
→ go version
go version go1.12.1 darwin/amd64

Reply:
ok

Reply:
go-datastore does not depend on `satori/go.uuid`. We use `google/uuid`.